diff --git a/node_modules/expo-firebase-core/android/build.gradle b/node_modules/expo-firebase-core/android/build.gradle
index 9bda09b..3f2a819 100644
--- a/node_modules/expo-firebase-core/android/build.gradle
+++ b/node_modules/expo-firebase-core/android/build.gradle
@@ -37,7 +37,7 @@ buildscript {
 
 // Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
+  archiveClassifier.set('sources')
   from android.sourceSets.main.java.srcDirs
 }
 
@@ -59,7 +59,7 @@ afterEvaluate {
 }
 
 android {
-  compileSdkVersion safeExtGet("compileSdkVersion", 31)
+  compileSdk safeExtGet("compileSdkVersion", 34)
 
   compileOptions {
     sourceCompatibility JavaVersion.VERSION_11
@@ -71,12 +71,12 @@ android {
   }
 
   defaultConfig {
-    minSdkVersion safeExtGet("minSdkVersion", 21)
-    targetSdkVersion safeExtGet("targetSdkVersion", 31)
+    minSdk safeExtGet("minSdkVersion", 21)
+    targetSdk safeExtGet("targetSdkVersion", 34)
     versionCode 10
     versionName '6.0.0'
   }
-  lintOptions {
+  lint {
     abortOnError false
   }
 }
